backend/src/api/app.ts

```diff
@@ -180,6 +180,10 @@ app.use(
 const authenticatedNoTermsRoute = express.Router();
 authenticatedNoTermsRoute.use(checkUserLoggedIn);
 authenticatedNoTermsRoute.get('/users/me', handlerToExpress(users.me));
+authenticatedNoTermsRoute.post(
+  '/users/me/generateApiKey',
```

Comment:
I'd suggest maybe changing this to be POST /api-keys and creating the `ApiKey` model to store the key and its permissions. This way, we are essentially creating service accounts and allowing for fine-grained access control for each API key in the future.